homeassistant/components/watchyourlan/config_flow.py

```diff
@@ -0,0 +1,67 @@
+"""Config flow for WatchYourLAN integration."""
+
+import logging
+from typing import Any
+
+from httpx import ConnectError, HTTPStatusError
+import voluptuous as vol
+from watchyourlanclient import WatchYourLANClient
+
+from homeassistant.config_entries import ConfigFlow, ConfigFlowResult
+from homeassistant.const import CONF_URL, CONF_VERIFY_SSL
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+# Schema for the user setup form
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_URL): str,
+        vol.Optional(CONF_VERIFY_SSL, default=False): bool,
+    }
+)
+
+
+class WatchYourLANConfigFlow(ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for WatchYourLAN."""
+
+    VERSION = 1
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle the initial step."""
+        errors = {}
+
+        if user_input is not None:
+            self._async_abort_entries_match()
```

Comment:
```suggestion
            self._async_abort_entries_match({})
```
We need to add what we are searching for